src/pages/createTarget/index.js

```diff
@@ -56,7 +60,7 @@ const CreateTarget = () => {
           <label htmlFor="topic">{t('newTarget.labels.topic')}</label>
           <Select
             name="topic"
-            options={[...TARGET_TOPICS]}
+            options={[...topics]}
```

Comment:
Instead of modifying the `Select` component (which was ok, no changes needed), you should map your `topics` array here to match the interface needed for the `options` prop of the `Select` component.

```js
option={topics.map(topic => topic.topic)}
```

or:

```js
option={topics.map(({ topic }) => topic)}
```